packages/builder/src/builderStore/store/frontend.js

```diff
@@ -1305,7 +1305,7 @@ export const getFrontendStore = () => {
     links: {
       save: async (url, title) => {
         const navigation = get(store).navigation
-        let links = [...navigation?.links]
+        let links = [...(navigation?.links ?? [])]
```

Comment:
There's a couple of these around the place:

The linter was complaining that safe navigation wasn't actually safe, which makes sense, because if `navigation` is undefined here, we can't spread it as it's not an interable.